migrations/2025-01-14-023145_media_filter/up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE local_user
+    ADD COLUMN hide_media boolean DEFAULT FALSE NOT NULL;
+
```

Comment:
Add an index for the new filtered column:

`CREATE INDEX idx_post_url_content_type ON post (url_content_type);`